db/data_migrations/20230614094504_remove_montenegro_from_quota_order_number_origins.rb

```diff
@@ -0,0 +1,14 @@
+Sequel.migration do
+  # IMPORTANT! Data migrations up block should be idempotent (reruns of up should produce the same effect)
+  # they may get re-run as part of data rollbacks but the rollback (down) function of the data migration will not get invoked
+
+  up do
+    if TradeTariffBackend.uk?
+      Sequel::Model.db[:quota_order_number_origins_oplog].where(quota_order_number_sid: 20_919, quota_order_number_origin_sid: 20_988, geographical_area_id: 'ME', operation_date: nil, oid: 8674, operation: 'C', geographical_area_sid: 348).delete
```

Comment:
The oid is a sequence that is specific to each database schema so you'll not want specify that.

I always like to add the filename since that indicates where the original was supposed to have come from (in this case its nil which is meaningful)